java/client/src/test/java/glide/api/models/TransactionTests.java

```diff
@@ -56,6 +62,56 @@ public void transaction_builds_protobuf_request() {
                         Info,
                         ArgsArray.newBuilder().addArgs(InfoOptions.Section.EVERYTHING.toString()).build()));
 
+        Map<String, Double> membersScores = new HashMap<String, Double>();
```

Comment:
same

Reply:
same here, the order of mappings in an immutable map are unspecified and cause these tests to fail sometimes and pass other times. We don't want flaky tests, so I'll put a comment here instead.